adoc/admin-security-certificates.adoc

```diff
@@ -461,6 +461,48 @@ kubectl replace -f oidc-gangway-cert.yaml
 kubectl rollout restart deployment/oidc-gangway -n kube-system
 ----
 
+** Replace the oidc root CA for `kube-apiserver`:
+
+. Copy custom oidc root CA which was used for `oidc-dex` and `oidc-gangway` to [path]`/etc/kubernetes/pki/oidc-ca.crt` on all CaaSP master nodes
++
+[source,bash]
+----
+ssh <USERNAME>@<MASTER_NODE_IP_ADDRESS/FQDN>
+sudo mv oidc-ca.crt /etc/kubernetes/pki/oidc-ca.crt
+----
+
+. If cluster wasn't bootstrapped yet, then update `oidc-ca-file` option in `<CLUSTER_INIT>/kubeadm-init.conf`
```

Comment:
Update `oidc-ca-file` to `/etc/kubernetes/pki/oidc-ca.crt`, then after cluster bootstrap, the dex&gangway server certificate would be signed by `pki/ca.crt` by skuba.

I worried this would not work~

Reply:
Fixed